src/data/working-groups.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name": "Example A",
+    "gitRemote": "https://github.com/EI2030/EI2030-example-working-group-a.git"
+  },
+  {
+    "name": "Group B",
+    "gitRemote": "https://github.com/EI2030/EI2030-example-working-group-b.git"
+  }
+]
```

Comment:
This is the entry point where we would add a new working group to the site.  All we should need to do is add it once here, and all markdown should be pulled in.

Reply:
Great thank you! :slightly_smiling_face: